app/templates/power-ups/topics.html

```diff
@@ -192,6 +192,10 @@ <h2 id="recoloring">Recoloring</h2>
   <p>For example, card badge icons are expected to be grey (#999) in their default state, but when a color is applied to the badge (such as ‘red’), the icon is passed <strong>?color=fff</strong>. This behavior means that the icon should be recolored such that the primary color used is white, ensuring the image will look good and have good usability on a red background.</p>
   <hr/>
 
+  <h2 id="timeouts">Timeouts</h2>
+  <p>Your Power-Up should respond within 1 second to all capability methods. If your method takes more than 1 second to complete, we will consider the callback as failed.</p>
```

Comment:
Hmm, we should probably take this opportunity to update this as well.

> Your Power-Up should respond within 1 second to all capability methods. If your method takes more than 1 second to complete, we will consider the callback as failed.

I would propose (although you should re-word however you like)

> When Trello makes a request to your Power-Up for a capability method and callbacks, it will only wait a fixed amount of time for a reply, before it will give up. Faster is better across all cases, and you should be aiming to reply within one second. Generally around five seconds is when Trello will stop waiting for a response, although some capabilities may wait slightly longer.
> Additionally, you should make sure that your manifest file and index connector can be loaded by Trello as quickly as possible. Trello will generally give up on these after 10 seconds, and you should aim to have these served within one second.